### DISCUSSION TITLE
Preset of slider shapes in editor

### DISCUSSION THREAD

**Discussion by hi-mei** [ppy/osu#13007] [Ideas] [answered]
Its quite annoying to deal with slider shapes especially the ones that are based on some kind of polygon or image (a star, triangle, square, heart etc). So it would be very usefull for mappers to have them already preset with perfect coordinates which would increase the quality of maps.

**Proposal designs of the feature:**
So same thing as current ctrl+shift+D work - add the same for Sliders.

**Comment by frenzibyte** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/issues/8773
